site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
-speeds:
-
-- The minimum clock speed, which results in the lowest power consumption.
-- The base clock speed, which results in a moderate level of power consumption
-  and heat generation. This tends to be the speed advertised in marketing
-  materials.
-- The maximum clock speed, sometimes marketed as "turbo boost." Running at this
-  speed causes unsustainable amounts of heating. It can only be used for short
-  periods of time, to satisfy bursts in demand for computing power.
-
-When a device's CPU utilization gets high, the device may increase clock speeds
-across its CPU cores, in an attempt to meet the demand for computing power. As
-the speeds exceed the base clock speed, the elevated power consumption increases
-the CPU's temperature. At some point, the device enters a
-[thermal throttling regime](https://en.wikipedia.org/wiki/Dynamic_frequency_scaling),
-where the CPU clock speed is reduced in order to bring the temperature down.
-
-As with CPU utilization, values from `0.0` (minimal) to `1.0` (boosted) are used
-to normalize clock speeds. A value of `0.5` represents the base clock speed for
-a given CPU.
-
-## Using the API
-
-The following code creates a `ComputePressureObserver`, configured with specific
-triggering thresholds, along with an associated callback function that may be
-invoked when a threshold is crossed. The callback is intentionally rate-limited,
-with the current implementation maxing out at one invocation per second for a
-page in the foreground, and once per ten seconds for background pages.
-
-Inside the callback function, your web application can respond to the changing
-utilization and clock speed levels by enabling or disabling CPU-intensive
-functionality.
-
-```javascript
-function callback(update) {
-  if (update.cpuSpeed > 0.5) {
-    // The CPU is running at faster than base speed.
-  } else {
-    // The CPU is running at normal or reduced speed.
-  }
-
-  if (update.cpuUtilization >= 0.9) {
-    // CPU utilization is over 90%.
-  } else if (update.cpuUtilization >= 0.75) {
-    // CPU utilization is over 75%.
-  } else if (update.cpuUtilization >= 0.5) {
-    // CPU utilization is over 50%.
-  } else {
-    // CPU utilization is under 50%.
-  }
-}
+Create the pressure observer by calling its constructor with a callback function to be run whenever
+there is a pressure update:
+
+```js
+let observer = new PressureObserver(callback, { sampleRate: 0.5 });
+```
+
+A sample rate, `sampleRate`, of 0.5, means that there will be updates at most every two seconds.
 
-const observer = new ComputePressureObserver(callback, {
-  // Thresholds divide the interval [0.0 .. 1.0] into ranges.
-  cpuUtilizationThresholds: [0.5, 0.75, 0.9],
-  // The minimum clock speed is 0.0, and the maximum speed is 1.0.
-  // 0.5 represents the base clock speed.
-  cpuSpeedThresholds: [0.5],
-});
+The sample rate requested cannot always be served by the system. It is restricted by the
+implementation.
```

Comment:
It is not clear what it means. Can you explain?

Reply:
I ll try.